dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue

```diff
@@ -3,7 +3,17 @@
     <p class="font-medium text-xl">Loading {{ humanizeString(DataTypeEnum.EutaxonomyNonFinancials) }} Data...</p>
     <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
   </div>
-  <div v-show="!waitingForData">
+  <div v-show="!waitingForData" data-test="multipleReportsBanner">
+    <ShowMultipleReportsBanner
+      data-test="multipleReportsBanner"
+      v-if="
+        extractedReportsAndReportingPeriods &&
+        extractedReportsAndReportingPeriods[0] &&
+        extractedReportsAndReportingPeriods[1]
```

Comment:
Why are we specifically checking for [0] and [1] ? Couldn't we check `extractedReportsAndReportingPeriods?.length > 1` ?

Reply:
also, by definition extractedReportsAndReportingPeriods has the length of 2, regardless of both entries.